spec/dummy/gemfiles/rails_6.1.gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: ../../..
   specs:
-    handcuffs (1.4.1)
+    handcuffs (2.0.0)
```

Comment:
This should end up at 
```suggestion
    handcuffs (2.1.0)
```
by the time we tag it.
But that can be in a followup PR for the actual version bump, if we want. (A PR that would finalize the release date in the CHANGELOG, too)